onedal/neighbors/neighbors.py

```diff
@@ -44,6 +44,12 @@
 
 
 class NeighborsCommonBase(metaclass=ABCMeta):
+    def _get_train_policy(self, queue, *data):
+        return _get_policy(queue, *data)
+
+    def _get_infer_policy(self, queue, *data):
+        return _get_policy(queue, *data)
```

Comment:
Could you please comment why you are duplicating this? The same method with different names...